torch/csrc/CudaIPCTypes.h

```diff
@@ -0,0 +1,61 @@
+#pragma once
+#include <c10/core/Allocator.h>
+
+#define CUDA_IPC_REF_COUNTER_FILE_SIZE 10000
+#define CUDA_IPC_WARN_AFTER_X_BLOCKS_IN_LIMBO 1000
+
+namespace torch {
+
+
+struct CudaIPCReceivedData {
```

Comment:
This is just a simple wrapper struct for extra type safety for shared pointers that are received data, is that right? Worth pointing out in the doc for this class.

Reply:
nit: mark this struct (and all the others) final